cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canols oil
```

Comment:
Looks like it is a spelling mistake.
Maybe you want to say "canola", instead of "canols" oil.